### DISCUSSION TITLE
tutorials/lift-off-part3/03-graphql-arguments

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#55] [course feedback]
# tutorials/lift-off-part3/03-graphql-arguments

https://www.apollographql.com/tutorials/lift-off-part3/03-graphql-arguments

**Comment by ZesarBlack** [comment]
totalmente entendible, me gustó

**Comment by acgriswold** [comment]
Great lesson!  Was pleasantly surprised to see the "Code Challenge!" restrict which lines to start typing on

**Comment by gustvoassuncao** [comment]
Awesome!

**Comment by amcooper** [comment]
Although I solved the code challenge, the solution isn't saved. My progress on this section remains stuck at 67% and my solution is gone when I return to the page. Any ideas? Thanks.

**Reply by lizhennessy** [reply]
hey there @amcooper ! happy to look into this. just to check, you've also completed the two multiple choice questions on the page, right?

**Reply by amcooper** [reply]
Yes. I'm assuming 67% means 2 out of 3: the two multiple choice items are acknowledged, while the third item, the code challenge, is not.

**Reply by amcooper** [reply]
Never mind, @lizhennessy . I did the code challenge one more time and the issue resolved itself :shrug:

**Reply by lizhennessy** [reply]
oh how strange @amcooper ! I'm glad it's working now - but do let us know if you encounter the bug again and we'll investigate further! 🕵️‍♀️
